### PR TITLE
case cleanup

### PR DESCRIPTION
**Pull request by snopoke**
@benrudolph 

cc @millerdev

### CODE REVIEW

**Comment by czue** [comment]
this looks great!

**Comment by benrudolph** [comment]
solid! i'll merge master in cause test failures are unrelated. feel free to self merge if i'm not on when the tests pass

**Comment by czue** [comment]
test fail looks real:
```
======================================================================
ERROR: test_dont_rebuild (corehq.apps.users.tests.retire.RetireUserTestCase)
Don't rebuild cases that are owned by other users
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/mock.py", line 1216, in patched
    arg = patching.__enter__()
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/mock.py", line 1275, in __enter__
    original, local = self.get_original()
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/mock.py", line 1263, in get_original
    "%s does not have the attribute %r" % (target, name)
AttributeError: <module 'casexml.apps.case.cleanup' from 'corehq/ex-submodules/casexml/apps/case/cleanup.pyc'> does not have the attribute 'rebuild_case'
```

File: corehq/apps/users/tests/retire.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def test_deleted_indices_removed(self):
         self.assertEqual(2, len(child.xform_ids))
 
 
-    @mock.patch("casexml.apps.case.cleanup.rebuild_case")
+    @mock.patch("casexml.apps.case.cleanup.rebuild_case_from_forms")
```
Comment (dimagimon) [inline corehq/apps/users/tests/retire.py]: E303 too many blank lines (2)

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by snopoke** [comment]
self merging

**Pull request merged**